tasks/request_from_archive/request_from_archive.py

```diff
@@ -623,12 +638,49 @@ def handler(event: Dict[str, Any], context: LambdaContext):  # pylint: disable-m
             context: This object provides information about the lambda invocation, function,
                 and execution env.
         Returns:
-            A dict with the value at 'payload' matching schemas/output.json
-                Combine with knowledge of CumulusMessageAdapter for other properties.
+            A dict matching schemas/output.json
         Raises:
             RestoreRequestError: An error occurred calling restore_object for one or more files.
             The same dict that is returned for a successful granule restore,
             will be included in the message, with 'success' = False for
             the files for which the restore request failed to submit.
     """
-    return run_cumulus_task(task, event, context)
+
+    # set the optional variables to None if not configured
+    try:
+        event["config"]["defaultRecoveryTypeOverride"] = event[
```

Comment:
My recommendation is still to place anything optional in its own section in the event, specifying the path it WOULD exist if present, then in-code pulling those properties in.

Reply:
merged your changes